src/components/stepforms/widgets/VariableInput/VariableInput.vue

```diff
@@ -170,6 +176,17 @@ export default class VariableInput extends Vue {
     }
   }
 
+  /**
+   * If the variable identifier is listed in the available variables display its value.
+   */
+  get variableValue() {
+    const matchingAvailableVariable = this.availableVariables.find(
+      aV => aV.identifier === this.variableIdentifier,
+    );
+    const value = matchingAvailableVariable?.value || '';
+    return Array.isArray(value) ? value.join(', ') : value;
+  }
```

Comment:
Good! It will also be useful in the dropdown that shows the variable and their value: https://github.com/ToucanToco/weaverbird/blob/master/src/components/stepforms/widgets/VariableInput/VariableInput.vue#L47